operator/src/test/java/org/keycloak/operator/PodTemplateE2EIT.java

```diff
@@ -0,0 +1,110 @@
+package org.keycloak.operator;
+
+import io.fabric8.kubernetes.client.KubernetesClientException;
+import io.quarkus.logging.Log;
+import io.quarkus.test.junit.QuarkusTest;
+import org.awaitility.Awaitility;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.keycloak.operator.v2alpha1.crds.Keycloak;
+
+import static java.util.concurrent.TimeUnit.MINUTES;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.keycloak.operator.v2alpha1.crds.KeycloakStatusCondition.HAS_ERRORS;
+import static org.keycloak.operator.v2alpha1.crds.KeycloakStatusCondition.READY;
+
+@QuarkusTest
+public class PodTemplateE2EIT extends ClusterOperatorTest {
+
+    @AfterEach
+    void afterEach() {
+        k8sclient
+                .resources(Keycloak.class)
+                .inNamespace(namespace)
+                .list()
+                .getItems()
+                .forEach(k -> k8sclient.resources(Keycloak.class).delete());
```

Comment:
what's the usage of this stream ? you are not using the `k` element.
you can replace all with : client.resources(Keycloak.class).delete()

Reply:
done 👍